<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>713</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>18</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="SmoothScrollArea" name="homePageLayout">
     <property name="styleSheet">
      <string notr="true">background: transparent; border: none</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>688</width>
        <height>713</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>24</number>
       </property>
       <property name="topMargin">
        <number>24</number>
       </property>
       <property name="rightMargin">
        <number>24</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="TitleLabel" name="TitleLabel">
           <property name="text">
            <string>今天</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TitleLabel" name="time_today_label">
           <property name="font">
            <font>
             <family>HarmonyOS Sans SC</family>
             <pointsize>18</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>11月45日 周日</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="lightColor" stdset="0">
            <color>
             <red>93</red>
             <green>93</green>
             <blue>93</blue>
            </color>
           </property>
           <property name="darkColor" stdset="0">
            <color>
             <red>209</red>
             <green>209</green>
             <blue>209</blue>
            </color>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="HorizontalFlipView" name="banner_view">
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>450</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>450</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="pagerLayout">
         <item>
          <widget class="HorizontalPipsPager" name="banner_pager">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item/>
           <item/>
           <item/>
           <item/>
           <item/>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="SubtitleLabel" name="SubtitleLabel_3">
         <property name="text">
          <string>推荐插件</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="rec_plugin_grid" rowstretch="0" columnstretch="0"/>
       </item>
       <item>
        <widget class="BodyLabel" name="BodyLabel">
         <property name="text">
          <string>这就到底了吗……(っ °Д °;)っ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="lightColor" stdset="0">
          <color>
           <red>125</red>
           <green>125</green>
           <blue>125</blue>
          </color>
         </property>
         <property name="darkColor" stdset="0">
          <color>
           <red>211</red>
           <green>211</green>
           <blue>211</blue>
          </color>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="blank">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SmoothScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HorizontalFlipView</class>
   <extends>QListWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HorizontalPipsPager</class>
   <extends>QListWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
